docs/data/introduction/licensing/licensing.md

```diff
@@ -0,0 +1,177 @@
+# Licensing
+
+<p class="description">MUI X is an open-core, MIT-licensed library. Purchase a commercial license for advanced features and support.</p>
+
+## MIT vs. commercial licenses
+
+MUI has been building MIT-licensed React components since 2014, and we are committed to the continued advancement of the open-source libraries.
+You can learn more about our stewardship ethos in [this document from our company handbook](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd).
+
+We offer commercial licenses to developers who need the most advanced features that cannot be easily maintained by the open-source community.
```

Comment:
"We offer commercial licenses" 

Isn't it a bit weird to say "offer" for something that we make people pay for ?

Reply:
No, it makes sense in this context. To "offer" something doesn't imply that it's free. I could offer to sell you my car, to give a random example.